### PR TITLE
Bookstore App Styling PR

### PR DESCRIPTION
**Pull request by Aseadd**
### **Tasks are done for the exercise**

- [ ] No linter error ✔️ 
- [ ] Gitflow configured well ✔️ 
- [ ] Apply best practices for JS and CSS ✔️ 
- [ ] Add Style for the App ✔️ 
- [ ] Style meet Zeplin design ✔️ 

book1
book2

### CODE REVIEW

File: src/components/styles.css [hunk 1]
```diff
@@ -379,6 +385,20 @@
   opacity: 1;
 }
 
+.error {
+  display: none;
+  margin-top: 20px;
+  text-align: center;
```
Comment (BenMukebo) [inline src/components/styles.css]: - [ ] Great job so far 👏🏻 However the Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). kindly change the background-color of your navbar, footer, and main page, ... to match the original design. Make sure the main page is aligned in the center, you need to fix the margin-left and margin-right and you need to add the missing part 👍🏻 

- Your Design:



- Original Design:

image

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Aseadd,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Professional README ✔️ 
- You have followed  JavaScript best practices. ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/styles.css [hunk 2]
```diff
@@ -7,27 +7,26 @@
...
+}
+
 .nav-bar {
   display: flex;
   flex-direction: row;
   width: 100%;
-  background: #c73e1d;
+  background-color: #fff;
   height: 4.938rem;
 }
```
Comment (AmaduKamara) [inline src/components/styles.css]: - [ ] Kindly justify the contents of the navbar to the center so your header style matches the original design just as the previous reviewer highlighted.

Hint:
Add this style to this nav-bar class `justify-content: center;` and things will be perfect as required 👍 

**Your Styles**
image

**The original design**
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
## Status: Requires Changes ♻️♻️♻️ 

Hi @Aseadd,

Good job so far on making the changes requested by the previous reviewer! 👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅  You used Gitflow correctly
✅ No linter error
✅ Good descriptive PR
✅ Your README is good
✅ App features work fine 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**